### PR TITLE
Fix CLI errors 

### PR DESCRIPTION
**Pull request by hargup**

### CODE REVIEW

**Comment by hargup** [comment]
cc @sahil-shubham , using these changes, I was able to build https://www.npmjs.com/package/@felvin-search/cli to a new version. Check it out

**Comment by sahil-shubham** [comment]
Still fails with the same error: 
image


I don't think we built it properly considering only the bin folder has been published to npm: (seeing the content of version 1.57.0)
image

File: package.json [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-  "main": "./src/index.ts",
+  "main": "./src/index.js",
   "publishConfig": {
     "access": "public",
     "main": "./dist/index.cjs.js"
   },
   "bin": {
     "felvin-cli": "bin/felvin-cli"
   },
-  "files": [
```
Comment (hargup) [inline package.json:14]: @sahil-shubham removing this did the trick.

**Comment by hargup** [comment]
@sahil-shubham The package is published now, and you can try it in in Instant apps directory by following commands:

-  `yarn add @felvin-search/cli`
- `./node_modules/@felvin-search/cli/bin/felvin-cli create-app`

Though `yarn create-app` is still not working. Can you check that?

**Pull request merged**